.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,42 @@
+name: Pre-commit auto-update
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Generate token
+        uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install pre-commit
+        run: pip install pre-commit
+
+      - name: Run pre-commit autoupdate
+        run: pre-commit autoupdate
+
+      - name: Create Pull Request
```

Comment:
You need to specify `base`.
![image](https://user-images.githubusercontent.com/31987104/134872113-f97c18eb-b33c-4a77-b1c8-4cfc1082ed9b.png)

Reply:
Why can't I use default behavior?

`Defaults to the branch checked out in the workflow.`
https://github.com/peter-evans/create-pull-request#action-inputs

Reply:
Because you didn't specify the branch in `actions/checkout`.

Reply:
Thanks! I got it.

Reply:
455ab04